api/tests/core/external/subcategory_suggestion_test.py

```diff
@@ -0,0 +1,40 @@
+from unittest import mock
+
+import pytest
+
+from pcapi.core.external import subcategory_suggestion
+from pcapi.core.external.subcategory_suggestion_backends import fixtures
+from pcapi.core.external.subcategory_suggestion_backends.subcategory_suggestion import SubcategorySuggestionBackend
+from pcapi.core.external.subcategory_suggestion_backends.test import TestBackend
+from pcapi.core.offerers import factories as offerers_factories
+
+
+@pytest.mark.usefixtures("db_session")
+class GetMostProbableSubcategoryIdsTest:
+    @mock.patch("pcapi.core.external.subcategory_suggestion.subcategory_suggestion_backend", TestBackend())
+    def test_get_most_probable_subcategory_ids(self):
+        category_ids = subcategory_suggestion.get_most_probable_subcategory_ids("Grosse teuf chez Solène")
+        assert category_ids[0] == "LIVRE_PAPIER"
+        assert category_ids[1] == "SPECTACLE_REPRESENTATION"
+        assert category_ids[2] == "ABO_PLATEFORME_VIDEO"
+
+    @mock.patch(
+        "pcapi.core.external.subcategory_suggestion.subcategory_suggestion_backend", SubcategorySuggestionBackend()
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏si tu veux vraiment tester avec un mock ici, j'aurais plutôt mocké la requête post, pas le backend